tests/test_examples.py

```diff
@@ -34,39 +36,72 @@ def get_file_paths():
     return test_files
 
 
-def create_checker(test_file, checker_name):
-    """Creates a test function from a test file, and a checker name.
-    test_file: The full path (string) to the file.
-    checker_name: The hyphenated checker name that should be detected.
-    An example of a valid checker_name would be: 'no-init-classes'
+def _get_full_pylint_json_output() -> List[Dict]:
+    output = subprocess.run(
+        ['pylint', '--reports=n',
+         '--rcfile=python_ta/.pylintrc',
+         '--output-format=json',
+         *get_file_paths()],
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE)
+
+    jsons_output = output.stdout.decode('UTF-8')
+    jsons_output = _strip_trailing_asni(jsons_output)
+    return json.loads(jsons_output)
+
+
+def _strip_trailing_asni(str_to_strip: str) -> str:
```

Comment:
I tried this out by running pytest both in the command line and in the Python Console without calling this function, and all of the tests passed. You can remove this function.

Reply:
Hm could I get some insight into what could be going wrong on my side? I still get the following error output from Pytest (if you scroll all the way right you can see the trailing ANSI code).
```python
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
C:\Users\craig\AppData\Local\Programs\Python\Python39\lib\json\__init__.py:346: in loads
    return _default_decoder.decode(s)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <json.decoder.JSONDecoder object at 0x000001760FDA9A60>
s = '[\r\n    {\r\n        "type": "convention",\r\n        "module": "C0103_invalid_name",\r\n        "obj": "",\r\n     ...n        "message": "\\"z\\" is not a valid mode for open.",\r\n        "message-id": "W1501"\r\n    }\r\n]\r\n\x1b[0m'
_w = <built-in method match of re.Pattern object at 0x000001760FD56AB0>
    def decode(self, s, _w=WHITESPACE.match):
        """Return the Python representation of ``s`` (a ``str`` instance
        containing a JSON document).
    
        """
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
        end = _w(s, end).end()
        if end != len(s):
>           raise JSONDecodeError("Extra data", s, end)
E           json.decoder.JSONDecodeError: Extra data: line 7725 column 1 (char 260860)
C:\Users\craig\AppData\Local\Programs\Python\Python39\lib\json\decoder.py:340: JSONDecodeError
```
I'm running this with
```python
>>> import pytest
>>> pytest.main(['tests/test_examples.py'])